src/WebJobs.Script.WebHost/Controllers/FunctionsController.cs

```diff
@@ -100,4 +100,33 @@ public FunctionsController(WebScriptHostManager scriptHostManager, WebHookReceiv
             return response;
         }
     }
+    
+    // Custom web hooks for extensions. 
+    public class HookController : ApiController
```

Comment:
I think this is feature creep. The goal of this work is to enable all the existing binding types that we have now, and you can do that without this new special webhook APIs which is a separate feature.